cmake/external_onnx.cmake

```diff
@@ -59,8 +59,23 @@ ExternalProject_Get_Property(ext_onnx SOURCE_DIR BINARY_DIR)
 
 set(ONNX_INCLUDE_DIR ${SOURCE_DIR}/onnx)
 set(ONNX_PROTO_INCLUDE_DIR ${BINARY_DIR}/onnx)
-set(ONNX_LIBRARY ${BINARY_DIR}/libonnx.a)
-set(ONNX_PROTO_LIBRARY ${BINARY_DIR}/libonnx_proto.a)
+if (WIN32)
+    set(ONNX_LIBRARY ${BINARY_DIR}/${CMAKE_BUILD_TYPE}/onnx.lib)
+    set(ONNX_PROTO_LIBRARY ${BINARY_DIR}/${CMAKE_BUILD_TYPE}/onnx_proto.lib)
+
+    ExternalProject_Add_Step(
```

Comment:
Do you need to copy the static library? L63-L64 says to link against the original.

Reply:
Approved since we can remove it later if needed.